ci/pipeline.yml

```diff
@@ -0,0 +1,89 @@
+# This is the pipeline to run the BDD tests on prod, pre-prod and dev. 
+# The triggers are manual for now, a ticket has been written to address this.
+resources:
+  - name: main
+    type: git
+    icon: github
+    check_every: never
+    webhook_token: gesture
+    source:
+      uri: https://github.com/ONSdigital/sml-catalogue.git
+      branch: main
+      username: ((github_access_token))
+      password: x-oauth-basic
+
+jobs:
+  - name: dev-bdd-tests
+    plan:
+      - get: main
+      - task: run-acceptance-tests
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: jasonbellons/python_3_slim
```

Comment:
Please can we avoid using images in personal repos. I'm happy to jump on a call to work out the best image/place to host it.

Reply:
amended to use the new selenium image. confirmed it now runs on all environments with this image